### ISSUE TITLE
Chore request: Rename the existing image/sprite tools

### ISSUE THREAD

**Issue by jonathanlinat** [joshuaskelly/quake-cli-tools#33]
Hi Joshua! 👋

I've been looking into our tool names and thought of a more accurate naming scheme:

- `image2spr` to `gif2spr`
- `spr2image` to `spr2gif`

Both tools currently work exclusively with GIF files. Adopting these names would reflect their actual functionality and help us establish a clearer naming convention across our toolkit. Sounds like a win-win, right? 🚀

**Comment by joshuaskelly** [comment]
Incorrect. They support all formats supported by Pillow.
https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html

In theory you could create an spr from a .PSD, or convert an spr into a .PDF!

**Comment by jonathanlinat** [comment]
Understood. So, those scripts must be adjusted based on your explanation. They are only referencing the GIF format in their codebase.

**Comment by joshuaskelly** [comment]
The code is actually fine. I created PNG and PDF versions of s_bubble.spr. What bit of code are you referencing?

**Comment by jonathanlinat** [comment]
Yep, you are absolutely right. The code is OK, but the instructions are not. There are only references to the GIF format.

![image](https://github.com/joshuaskelly/quake-cli-tools/assets/14064112/a9080dbe-7003-4103-bba4-82447588cd82)

As you can see, I do believe these need to be adjusted to ensure the developers and mappers understand that this script is working with all the formats provided by Pillow, as you mentioned.

Does that make sense?
